source/tests/system/rfmx_expect_macros.h

```diff
@@ -24,11 +28,12 @@
     EXPECT_EQ(expected_error, response.status());       \
   }
 
-#define EXPECT_SUCCESS(session, response) \
-  if (1) {                                \
-    EXPECT_RESPONSE_SUCCESS(response)     \
-    CHECK_ERROR(session);                 \
-  }
+#define EXPECT_SUCCESS(session_, response_)     \
+  ([this](auto& session, auto& response) {      \
```

Comment:
Ah, thanks for fixing this up! I spent a little time trying to figure it out but gave up on it.